make/Main.gmk

```diff
@@ -109,6 +109,17 @@ ALL_TARGETS += generate-exported-symbols
 # When creating a BUILDJDK, the java targets have already been built and copied
 # into the buildjdk so no need to generate sources.
 ifneq ($(CREATING_BUILDJDK), true)
+  GENERATE_BUILDJDK_SOURCES := true
+else ifeq ($(OPENJDK_TARGET_OS)+$(COMPILE_TYPE), windows+cross)
```

Comment:
Wouldn't it be simpler to just surround the original `ifneq` with a condition like this. I.e., if windows+cross always set generate_build_jdk_sources = true, otherwise execute the original condigion.

Reply:
> Wouldn't it be simpler to just surround the original `ifneq` with a condition like this. I.e., if windows+cross always set generate_build_jdk_sources = true, otherwise execute the original condigion.

I'm not sure I follow: the original condition doesn't have GENERATE_BUILDJDK_SOURCES since it is a new variable I have introduced. If we don't execute the if block from the original check, then line 767 in the new file will fail.